frontend/src/views/QuizCreateView.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+  <div class="container">
+    <!-- 문제 내용 입력 컴포넌트 -->
+    <div class="content-field">
+      <label for="content" class="content">문제 내용</label>
+      <textarea type="text" :value="content" placeholder="문제를 입력하세요" @input="handleContentInput"
+        class="content-input" />
+    </div>
+    <hr />
+
+    <!-- 선택지 입력 컴포넌트들 -->
+    <div v-for="(option, index) in options" :key="index" class="option-field">
+      <div class="option-input-wrapper">
+        <!-- 선택지 입력란 -->
+        <input type="radio" :name="'option'" :value="index + 1" v-model="answer" :id="'option' + index" hidden />
+        <label :for="'option' + index" class="answer-button" :class="{ 'selected': answer === index + 1 }">
+          <i :class="answer === index + 1 ? 'bi bi-check-square' : 'bi bi-square'"></i>
+        </label>
+        <quiz-create-option :value="option" :placeholder="'선택지 ' + (index + 1) + '을 입력하세요'"
+          @input="handleOptionInput(index, $event)" />
```

Comment:
- 컴포넌트를 in-Dom 템플릿에서 `kebab-case`로 사용하는 경우 셀프 클로징 컴포넌트로 사용하면 안돼
참고
[Vue Style Guide 우선순위 B](https://ko.vuejs.org/style-guide/rules-strongly-recommended.html#self-closing-components)
```suggestion
        <quiz-create-option :value="option" :placeholder="`선택지 ${index + 1} 을 입력하세요`"
          @input="handleOptionInput(index, $event)" ></quiz-create-option>
```

Reply:
정정했습니다.